pyartcd/pyartcd/pipelines/prepare_release.py

```diff
@@ -148,9 +152,22 @@ async def run(self):
             advisory_type = "RHEA" if is_ga else "RHBA"
             for ad in advisories:
                 if advisories[ad] < 0:
-                    advisories[ad] = self.create_advisory(advisory_type=advisory_type, art_advisory_key=ad)
-            await self._slack_client.say_in_thread(f"Advisories created with release date {self.release_date}")
-
+                    if ad == "advance":
+                        # Set release date to one week before
+                        # Eg one week before '2024-Feb-07' should be '2024-Jan-31'
```

Comment:
I am a little confused about the release date, the GA date is Feb 27, we prepare advance release 2 weeks before(Feb13) so will set the release date(Feb20) one week before GA, why need calculate again

Reply:
thanks for the comment, lets discuss in the thread